modules/kube-apiserver-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="kube-apiserver-operator_{context}"]
+= Kubernetes API Server Operator
+
+[discrete]
+== Purpose
+
+The Kubernetes API Server Operator manages and updates the Kubernetes API server
+deployed on top of {product-title}.
```

Comment:
Refer to the README, add 'The operator is based on OpenShift library-go framework and it is installed via Cluster Version Operator (CVO)' to the end of the line.